### DISCUSSION TITLE
Documentation error

### DISCUSSION THREAD

**Discussion by kthxbai0815** [jcorporation/myMPD#767] [General]
Hello!
I recently discovered mympd, I'm using the docker version on an Odroid HC1 running Armbian. Well done, I really like this software so far.

Reading the documentation I came across a 404 - file not found error, the keyboard-shortcuts section is gone.
Cheers!

**Comment by jcorporation** [comment]
Thanks for the hint, link is now removed. Keyboard shortcuts are documented in the help dialog.
